pkg/utils/getEnv.go

```diff
@@ -25,3 +25,15 @@ func GetWatchNamespace() string {
 	ns, _ := os.LookupEnv("WATCH_NAMESPACE")
 	return ns
 }
+
+// GetEnableOpreqWebhook check if enable the webhook for the OperandRequest
+func GetEnableOpreqWebhook() bool {
+	enable, ok := os.LookupEnv("ENABLE_OPREQ_WEBHOOK")
```

Comment:
Should we update the csv in `cs-operator` for webhook about the env `ENABLE_OPREQ_WEBHOOK`?

Reply:
Yes, I will create a PR for it after this PR is merged. Then I can update the image digest with the same PR.

Reply:
Got it.